app/schema/widgets/checkbox_group_widget.py

```diff
@@ -0,0 +1,48 @@
+from app.schema.widget import Widget
+from flask import render_template
+from app.libs.utils import ObjectFromDict
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class CheckboxGroupWidget(Widget):
+    def __init__(self, name):
```

Comment:
This doesn't seem to differ from the base class method? Why is it restated here?